packages/client/src/pages/Profile/elements/ProfileInfo/ProfileInfo.tsx

```diff
@@ -0,0 +1,97 @@
+import { FC, useEffect, useState } from 'react'
+import styles from './profileInfo.module.css'
+import Title from '@components/Title'
+import Avatar from '@pages/Profile/elements/Avatar'
+import Form from '@components/Form'
+import { fieldsConfig } from '@utils/validation/fieldsConfig'
+import { omit } from 'lodash'
+import { useCheckAuthentication } from '@utils'
+import { UserDTO } from '@api/auth/types'
+import { resourcesApi } from '@api'
+import { updateProfile } from '@services/profile'
+import { ErrorResponse } from '@types'
+
+export const ProfileInfo: FC = () => {
+  const [editMode, setEditMode] = useState(false)
+  const [avatarUrl, setAvatarUrl] = useState('')
+  const [isLoading, setIsLoading] = useState(true)
+  const toggleEditMode = (e?: React.MouseEvent<HTMLButtonElement>) => {
+    e?.preventDefault()
+    setEditMode(prevEditMode => !prevEditMode)
+  }
+  const [userData, setUserData] = useCheckAuthentication()
+  const handleSaveProfile = async (
+    data: Omit<UserDTO, 'id' | 'display_name' | 'avatar'>
+  ) => {
+    if (editMode && userData && 'display_name' in userData) {
+      const newProfileInfo = await updateProfile({
+        ...data,
+        display_name: userData?.display_name,
+      })
+      if (typeof setUserData === 'function') {
+        setUserData(newProfileInfo || null)
+        toggleEditMode()
+      }
+    }
+  }
+
+  useEffect(() => {
+    const fetchAvatar = async () => {
+      setIsLoading(true)
+      if (!userData) {
+        return
+      }
+      try {
+        if ('avatar' in userData && userData.avatar) {
```

Comment:
[Исправление]
Тут не нужно проверять наличие ключа в объекте. Если `avatar` нет в `userData`, то `userData.avatar` будет `undefined`
```suggestion
        if (userData.avatar) {
```